### ISSUE TITLE
Naming/ConstantName ignores constants with frozen ranges

### ISSUE THREAD

**Issue by iMacTia** [rubocop-hq/rubocop#6808]
The issue is with the `Naming/ConstantName` cop and it's the same as this ticket:
https://github.com/rubocop-hq/rubocop/issues/5155

The issue was reportedly fixed in https://github.com/rubocop-hq/rubocop/pull/5157, but it seems like it's still affecting frozen Range constants.

**NOTE: this only applies when the constant value is a range (see example below).**

--------

## Expected behavior

Rubocop should detect `Naming/ConstantName` offences when a constant Range is frozen.

## Actual behavior

Rubocop doesn't detect `Naming/ConstantName` offences when a constant Range is frozen.

## Steps to reproduce the problem

Run latest version of Rubocop on code having the following line:

```ruby
MyInvalidConstantName = (400...500).freeze
```

## RuboCop version

This was tested with all rubocop version from 0.58.0 to 0.65 on macOS Mojave.
ALL versions had the same issue.
Below is only the latest version.

```
$ [bundle exec] rubocop -V
0.65.0 (using Parser 2.6.0.0, running on ruby 2.5.1 x86_64-darwin17)
```

**Comment by iMacTia** [comment]
EDIT: After additional research, it turned out this is only an issue with ranges. I've updated the issue description.

**Comment by Drenmi** [comment]
Thanks for the report @iMacTia! I saw this code in Faraday, and suspected there might be something going on. 🙂 Will have a look tonight.

**Comment by iMacTia** [comment]
Thank you everyone for quickly addressing this 🎉!
